src/install.ts

```diff
@@ -57,6 +56,13 @@ class UI {
     });
     return Promise.resolve(result); // Thenable to real promise.
   }
+  localize(message: string, ...args: Array<string|number|boolean>): string {
```

Comment:
Ah, nice! I forgot about this. Looks like the person who contributed this support to node-clangd only cared about having it in coc-clangd :laughing: 

This also partially answers my question about the vscode engine version: per the original discussion at https://github.com/clangd/vscode-clangd/issues/560, localization requires 1.73.

(You can add "Fixes https://github.com/clangd/vscode-clangd/issues/560" to the commit message.)